docs/README.md

```diff
@@ -9,8 +9,8 @@ Yarn >= 1.22.4
 
 ## Build the doc site
 
-1. In the command line, make sure you are in the docs folder: `cd docs`.
-1. Run `make docs`. This launches a preview of the docs website at `http://localhost:3002/docs/grafana/latest/` which will refresh automatically when changes to content in the `sources` directory are made.
+1. Onn the command line, first change to the docs folder: `cd docs`.
```

Comment:
Fix typo, "Onn"

Reply:
Well gee, that was embarrassing.  Amended.